okteto.yml

```diff
@@ -19,8 +19,8 @@ build:
     target: dev
 
 deploy:
-  - helm upgrade --install postgresql postgresql/postgresql-11.6.2.tgz -f postgresql/values.yml
-  - helm upgrade --install kafka kafka/kafka-17.1.0.tgz -f kafka/values.yml
+  - helm upgrade --install postgresql oci://registry-1.docker.io/bitnamicharts/postgresql -f postgresql/values.yml --version 13.4.4
```

Comment:
If we install from docker.io, we can delete the kafka and postgresql *.tgz. local charts.
We did it that way because bitnami URLs failed to often, but this shouldn't be the case now with docker.io

Reply:
yeah, they are deleted as part of the PR (I'm assuming you mean the tgz files?).

Reply:
oh, I didn't see it, great!